src/meinberlin/meinberlin/static/js/Packages/Meinberlin/IdeaCollection/IdeaCollection.ts

```diff
@@ -102,6 +106,22 @@ export var proposalDetailColumnDirective = (
         link: (scope, element, attrs, column : AdhMovingColumns.MovingColumnController) => {
             column.bindVariablesAndClear(scope, ["processUrl", "proposalUrl"]);
             adhPermissions.bindScope(scope, () => scope.proposalUrl && AdhUtil.parentPath(scope.proposalUrl), "proposalItemOptions");
+            adhPermissions.bindScope(scope, () => scope.proposalUrl && AdhUtil.parentPath(scope.proposalUrl), "versionOptions");
+            scope.delete = () => {
+                var proposalClass = RIGeoProposal;
+                if (scope.isKiezkasse) {
+                    proposalClass = RIKiezkasseProposal;
+                } else if (scope.isBuergerhaushalt) {
+                    proposalClass = RIBuergerhaushaltProposal;
+                }
+                // FIXME: translate
+                if ($window.confirm("Do you really want to delete this?")) {
+                    adhHttp.hide(AdhUtil.parentPath(scope.proposalUrl), proposalClass.content_type)
+                        .then(() => {
+                            adhTopLevelState.goToCameFrom("/");
+                        });
+                }
+            };
```

Comment:
I am not sure whether this shouldn't be in the proposal directive. This would at least have been my initial choice. But maybe it is actually better to have it here because (1) you need to know the workbench context in order to know where to redirect after successful deletion and (2) this is not really specific to a proposal.